oidc-controller/api/core/oidc/provider.py

```diff
@@ -0,0 +1,47 @@
+from jwkest.jwk import rsa_load, RSAKey, KEYS
+
+from pyop.authz_state import AuthorizationState
+from pyop.provider import Provider
+from pyop.storage import RedisWrapper
+from pyop.subject_identifier import HashBasedSubjectIdentifierFactory
+from pyop.userinfo import Userinfo
+
+from ..config import settings
+
+db_uri = settings.REDISDB_URL
+
+
+signing_key = RSAKey(key=rsa_load("signing_key.pem"), use="sig", alg="RS256")
+signing_keys = KEYS().append(signing_key)
+
+configuration_information = {
+    "issuer": "https://host.docker.internal:5201",
+    "authorization_endpoint": "https://host.docker.internal:5201/authorization",
+    "token_endpoint": "https://host.docker.internal:5201/token",
+    "jwks_uri": "https://host.docker.internal:5201/.well-known/openid-configuration/jwks",
+    "response_types_supported": ["code", "id_token token"],
+    "id_token_signing_alg_values_supported": [signing_key.alg],
+    "response_modes_supported": ["fragment", "query"],
+    "subject_types_supported": ["public", "pairwise"],
+    "grant_types_supported": ["authorization_code", "implicit"],
+    "claim_types_supported": ["normal"],
+    "claims_parameter_supported": True,
+    "claims_supported": ["sub", "name", "given_name", "family_name"],
+    "request_parameter_supported": True,
+    "request_uri_parameter_supported": False,
+    "scopes_supported": ["openid", "profile"],
+}
```

Comment:
Is this a standard configuration for the library? Unless there are parameters that should NOT be changed, we should extract this so we can provide it to the application at runtime.

A couple of examples:
- the host names will not work outside of docker, they will likely need to be populated with the URLs for the hosted service
- supported claims `name`, `given_name`, `family_name` may not be supported - this will depend on whether the credential has claims with those attribute names

Reply:
parameterized,
removed, 

I am not familiar enough with these to be confident in them. should be reviewed after an integration. i'll leave a comment.